src/main/scala-2.11/markets/clearing/engines/CDAMatchingEngine.scala

```diff
@@ -86,7 +86,7 @@ class CDAMatchingEngine(askOrdering: PriceOrdering[AskOrder],
         if (residualQuantity < 0) {
           val totalMatch = TotalMatch(askOrder, incoming, price)
           // add residualOrder back into orderBook!
-          val residualOrder = askOrder.split(-residualQuantity)
+          val (_, residualOrder) = askOrder.split(-residualQuantity)
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Field name does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1537948179/issues/source?bid=2921361&fileBranchId=3014278#l89)